audit/format.go

```diff
@@ -127,6 +128,11 @@ func (f *AuditFormatter) FormatRequest(ctx context.Context, w io.Writer, config
 		},
 	}
 
+	var zt time.Time
+	if auth.IssueTime != zt {
+		reqEntry.Auth.TokenIssueTime = auth.IssueTime.Format(time.RFC3339)
```

Comment:
Is there precedent for stringifying times before auditing?  Does it work if you don't?  I'd prefer to defer formatting as late as possible.  This also means if we decide that times don't need to be redacted, it's easy to not to do so.

Reply:
Sorry, I should clarify: I know we do it for some "envelope" fields like AuditResponseEntry.Time, I mean for the actual Auth/Request/Response fields that go through HashAuth/HashRequest/HashResponse.  But it looks like you're doing this after redaction, so it's moot.  Which is a bit questionable, mind you.  It's probably ok, and seems to achieve the desired effect. but it's a risky practice to get into.

Reply:
My concern was with parseability by Splunk and other systems.  I assume unix time would also work but it makes the audit log not human readable, a problem I didn't want to make worse.  Re doing after redaction, that's probably just an oversight on my part, but these fields if redacted are pretty much useless.  In particular, due to the small domain size of ttl, its pretty easy to reverse the hmac via a dictionary if you have the ability to create tokens anyway.

Reply:
Oh, I agree that we probably don't want these fields redacted.  I just want redaction (and the decision as to whether to redact) to be in the same place as much as possible.  I also want times rendered with RFC3339, as you were doing.